platform/ios/CHANGELOG.md

```diff
@@ -32,7 +39,6 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 * Added `MGLLoggingConfiguration` and `MGLLoggingBlockHandler` that handle error and fault events produced by the SDK. ([#13235](https://github.com/mapbox/mapbox-gl-native/pull/13235))
 * Fixed random crashes during app termination. ([#13367](https://github.com/mapbox/mapbox-gl-native/pull/13367))
 * Fixed a crash when specifying MGLShapeSourceOptionLineDistanceMetrics when creating an MGLShapeSource. ([#13543](https://github.com/mapbox/mapbox-gl-native/pull/13543))
```

Comment:
`MGLShapeSourceOptionLineDistanceMetrics` & `MGLShapeSource`

Reply:
Have updated in the cherry-pick. We should address this (in master) in a subsequent PR.